### PR TITLE
SEO-176113-Pages-With-Redirects-WinForms1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on links with redirects.

Kind regards,
Beryl

### CODE REVIEW

File: WindowsForms/GridGrouping/Navigation-Bar.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ Me.gridGroupingControl1.RecordNavigationControl.ForeColor = Color.White
 ![Navigation-Bar_img8](Navigation-Bar_images/Navigation-Bar_img8.png)
 
 ## VisualStyles
-The style of the `NavigationBar` can be changed by using the [GridOfficeScrollBar](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.Grouping.GridGroupingControl.html#Syncfusion_Windows_Forms_Grid_Grouping_GridGroupingControl_GridOfficeScrollBars) property which automatically changes the visual style. Check [Scrolling](http://help.syncfusion.com/windowsforms/gridgrouping/getting-started) topic for more information on this.
+The style of the `NavigationBar` can be changed by using the [GridOfficeScrollBar](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.Grouping.GridGroupingControl.html#Syncfusion_Windows_Forms_Grid_Grouping_GridGroupingControl_GridOfficeScrollBars) property which automatically changes the visual style. Check [Scrolling](https://help.syncfusion.com/windowsforms/gridgrouping/getting-started) topic for more information on this.
```
Comment (Yvone-Atieno) [inline WindowsForms/GridGrouping/Navigation-Bar.md:186]: Update scrolling link
image

File: WindowsForms/GridGrouping/Grouping.md [hunk 2]
```diff
@@ -1138,7 +1138,7 @@ The following events are used to customize the grouping in GridGroupingControl.
 * [GroupCollapsing](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.Grouping.GridGroupingControl.html) - Occurs before a group is collapsed.
 * [GroupCollapsed](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.Grouping.GridGroupingControl.html) - Occurs after a group is collapsed.
```
Comment (Yvone-Atieno) [inline WindowsForms/GridGrouping/Grouping.md:1139]: Update the below link - https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.Grouping.GridGroupingControl.html#Syncfusion_Windows_Forms_Grid_Grouping_GridGroupingControl_GroupCollapsed

File: WindowsForms/GridGrouping/Grouping.md [hunk 3]
```diff
@@ -1138,7 +1138,7 @@ The following events are used to customize the grouping in GridGroupingControl.
 * [GroupCollapsing](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.Grouping.GridGroupingControl.html) - Occurs before a group is collapsed.
```
Comment (Yvone-Atieno) [inline WindowsForms/GridGrouping/Grouping.md:1138]: Update the correct link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/GridGrouping/Exporting.md [hunk 4]
```diff
@@ -341,7 +341,7 @@ Exported preview row in excel,
 ![Exporting_img8](Exporting_images/Exporting_img8.png)
 
 ### Exporting Sparkline Column
-The Sparkline column or cells in the GridGroupingControl can be exported to Excel by defining the spark line groups in the excel sheet. It can be achieved by using the [ISparklineGroups](https://docs.microsoft.com/en-us/dotnet/api/microsoft.office.interop.excel.isparklinegroups?view=excel-pia) interface.
+The Sparkline column or cells in the GridGroupingControl can be exported to Excel by defining the spark line groups in the excel sheet. It can be achieved by using the [ISparklineGroups](https://learn.microsoft.com/en-us/dotnet/api/microsoft.office.interop.excel.isparklinegroups?view=excel-pia) interface.
 
 It caches the `SparklineGroup` that needs to be added to the Excel sheet. The spark lines will be appear once the user select the data range and the location range. Data range and reference range for the Sparkline in Excel can be determined and assigned in [QueryImportExportCellInfo](https://help.syncfusion.com/cr/windowsforms/Syncfusion.GridExcelConverter.GridExcelConverterBase.html) event.
```
[the remark this answers is not in the document]
Comment (berylouma) [reply WindowsForms/GridGrouping/Exporting.md:346]: Updated

File: WindowsForms/GridGrouping/Navigation-Bar.md (hunk 1 above)
Comment (berylouma) [reply WindowsForms/GridGrouping/Navigation-Bar.md:186]: Updated

File: WindowsForms/GridGrouping/Grouping.md (hunk 2 above)
Comment (berylouma) [reply WindowsForms/GridGrouping/Grouping.md:1139]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
> fine.

**Pull request merged**